### PR TITLE
[DO Not merge]Testing build update to use 32-bit version of dotnet.exe

### PR DESCRIPTION
**Pull request by pragnagopa**
Fixes #2809

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 $currentDir = Split-Path -Parent $MyInvocation.MyCommand.Path
 $buildOutput = Join-Path $currentDir "buildoutput"
+$dotnetx86 = "C:\Program Files (x86)\dotnet\dotnet.exe"
```
Comment (fabiocav) [inline build.ps1]: Would be a good idea to use the `ProgramFiles(x86)` environment variable here.

File: build.ps1 [hunk 2]
```diff
@@ -37,11 +40,11 @@ function CrossGen([string] $runtime, [bool] $isSelfContained, [string] $publishT
     DownloadNupkg "https://www.nuget.org/api/v2/package/Microsoft.Build.Utilities.Core/15.1.1012" @("lib\netstandard1.3\Microsoft.Build.Utilities.Core.dll")  @("$selfContained")
 
     # Publish self-contained app with all required dlls for crossgen
-    dotnet publish .\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj -r $runtime -o "$selfContained" -v q /p:BuildNumber=$buildNumber    
+    & $dotnetx86 publish .\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj -r $runtime -o "$selfContained" -v q /p:BuildNumber=$buildNumber
```
Comment (fabiocav) [inline build.ps1]: We shouldn't need to change all the publish/build steps, just the test runs, which are currently defined in the appveyor.yml file (would be a good idea to break that off). Right?

File: appveyor.yml [hunk 3]
```diff
@@ -30,13 +30,26 @@ for:
...
   max_jobs: 1
   clone_folder: c:\azure-webjobs-sdk-script
   install:
   - ps: >-
       $env:CommitHash = "$env:APPVEYOR_REPO_COMMIT"
 
       Install-Product node 8.4.0 x86
+  install_dotnet_script_x86:
+  - ps: |
+      $dotnetversion = 2.1.200
```
Comment (pragnagopa) [inline appveyor.yml]: @fabiocav - x86 version of dotnet on appveyor machines is too old. Is there a specific version we want to install?

Comment (fabiocav) [reply appveyor.yml]: Whatever version is the latest released. We'll soon be updating this when the DI changes land.

**Comment by mathewc** [comment]
Is this PR still active, or can it be closed?

**Pull request closed without merging**